nodes/sw-wifi/config.txt
hostname mySwitch
!
!
vlan 1001-1016
!
vtp mode transparent
!
interface FastEthernet0/1
 switchport access vlan 1001
 switchport mode dot1q-tunnel
 shutdown
 power inline auto

 no shutdown
no cdp enable
!
interface FastEthernet0/2
 switchport access vlan 1001
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/3
 switchport access vlan 1002
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/4
 switchport access vlan 1002
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/5
 switchport access vlan 1003
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/6
 switchport access vlan 1003
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/7
 switchport access vlan 1004
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/8
 switchport access vlan 1004
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/9
 switchport access vlan 1005
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/10
 switchport access vlan 1005
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/11
 switchport access vlan 1006
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/12
 switchport access vlan 1006
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/13
 switchport access vlan 1007
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/14
 switchport access vlan 1007
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/15
 switchport access vlan 1008
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/16
 switchport access vlan 1008
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/17
 switchport access vlan 1009
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/18
 switchport access vlan 1009
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/19
 switchport access vlan 1010
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/20
 switchport access vlan 1010
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/21
 switchport access vlan 1011
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/22
 switchport access vlan 1011
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/23
 switchport access vlan 1012
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable

interface FastEthernet0/24
 switchport access vlan 1012
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/25
 switchport access vlan 1013
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/26
 switchport access vlan 1013
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/27
 switchport access vlan 1014
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/28
 switchport access vlan 1014
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/29
 switchport access vlan 1015
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/30
 switchport access vlan 1015
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/31
 switchport access vlan 1016
 switchport mode dot1q-tunnel
 shutdown
 power inline auto
 no shutdown
no cdp enable
!
interface FastEthernet0/32
 switchport access vlan 1016
 switchport mode dot1q-tunnel
 power inline never
 no shutdown
no cdp enable
!
interface FastEthernet0/48
 switchport trunk encapsulation dot1q
 switchport mode trunk
!
!
! 
interface vlan1
 ip address 192.168.255.1 255.255.255.0
 no shutdown
!
!
snmp-server group groupTPWifi v3 auth read vieWifi write vieWifi
snmp-server user userWifi groupTPWifi v3 auth md5 passWifi
snmp-server view vieWifi iso included
!


nodes/srv-poe/html/config.php
<?php

$USER = "userWifi";
$PASSWORD = "passWifi";

$HOSTS = [
	'sw_wifi' => [ 	'ip' => "10.108.135.252",
					'oidState' => ['1.3.6.1.2.1.2.2.1.7'],	  // AdminStatus
					'valueOn' => "1",
					'portOffset' => 0],
];

$NODES = [

	'Point d\'accès Ubiquiti' => [
		'AP1' => ['parent' => 'sw_wifi', 'port' => 1],
		'AP2' => ['parent' => 'sw_wifi', 'port' => 2],
		'AP3' => ['parent' => 'sw_wifi', 'port' => 3],
		'AP4' => ['parent' => 'sw_wifi', 'port' => 4],
		'AP5' => ['parent' => 'sw_wifi', 'port' => 5],
		'AP6' => ['parent' => 'sw_wifi', 'port' => 6],
		'AP7' => ['parent' => 'sw_wifi', 'port' => 7],
		'AP8' => ['parent' => 'sw_wifi', 'port' => 8],			
	],
	'Ponts Ubiquiti (1-4)' => [
		'BR1A' => ['parent' => 'sw_wifi', 'port' => 9],
		'BR1B' => ['parent' => 'sw_wifi', 'port' => 10],
		'BR2A' => ['parent' => 'sw_wifi', 'port' => 11],
		'BR2B' => ['parent' => 'sw_wifi', 'port' => 12],
		'BR3A' => ['parent' => 'sw_wifi', 'port' => 13],
		'BR3B' => ['parent' => 'sw_wifi', 'port' => 14],
		'BR4A' => ['parent' => 'sw_wifi', 'port' => 15],
		'BR4B' => ['parent' => 'sw_wifi', 'port' => 16],	
	],
	

];
?>